schema/base/land.yaml

```diff
@@ -0,0 +1,74 @@
+---
+"$schema": https://json-schema.org/draft/2020-12/schema
+title: Land Schema
+description: Land
+type: object
+properties:
+  geometry:
+    unevaluatedProperties: false
+    oneOf:
+      - "$ref": https://geojson.org/schema/Point.json
+      - "$ref": https://geojson.org/schema/LineString.json
+      - "$ref": https://geojson.org/schema/MultiLineString.json
+      - "$ref": https://geojson.org/schema/Polygon.json
+      - "$ref": https://geojson.org/schema/MultiPolygon.json
+  properties:
+    unevaluatedProperties: false
+    allOf:
+      - "$ref": ./defs.yaml#/$defs/propertyContainers/osmPropertiesContainer
+      - "$ref": ../defs.yaml#/$defs/propertyContainers/overtureFeaturePropertiesContainer
+      - "$ref": ../defs.yaml#/$defs/propertyContainers/levelContainer
+    required:
+      - class
+      - subclass
+    properties:
+      class:
+        description: Broad class of landcover, such as forest, glacier, grass, or a physical feature.
+        default: [land]
+        type: string
+        enum:
+           - forest
+           - glacier
+           - grass
+           - land
+           - physical
+           - reef
+           - rock
+           - sand
+           - shrub
+           - tree
+           - wetland
+      subclass:
```

Comment:
Not sure how accurate we want to be in validation at this stage of Base Theme introduction (but our schema is about providing some guarantee in space of properties validity vs osm). Some of subclass values have a sense only for specific classes - no?. It could be then beneficial to have schema for subclass constructed through conditional values of subclass depending on value of class.

Reply:
My guess is it's tough to get in for the upcoming release but I think this is right since some subclasses should only work with some classes and the schema should capture that.